addons/default/visiosoft/defaultadmin-theme/src/Listener/AddGsmFilter.php

```diff
@@ -52,16 +53,36 @@ protected function addGsmFilter(UserTableBuilder $builder)
         $builder->setColumns([
                 'email',
                 'gsm_phone',
+                'created_at' => [
+                    'value' => 'entry.created_at'
+                ],
                 'status' => [
                     'value' => 'entry.status_label',
                 ],]
         );
-        $c = Collection::make([
-            ['heading' => 'anomaly.module.users::field.email.name'],
-            ['heading' => 'visiosoft.module.profile::field.gsm_phone.name'],
-            ['heading' => 'anomaly.module.users::field.status.name'],
+        $builder->setOptions([
+            'order_by' =>
+                [
+                    'email' => 'DESC'
+                ],
         ]);
-        $builder->getTable()->setHeaders($c);
+
+        $collection = new Collection();
```

Comment:
Too many duplicate header. İnstead foreach can be used for types and collections headers.